### PR TITLE
Update README.md task 3

### PR DESCRIPTION
**Pull request by jalas167**

### CODE REVIEW

File: tasks/3-parallel-file-conversion/README.md [hunk 1]
```diff
@@ -54,16 +54,16 @@ base64 converted, chunked, and sent as strings within the task, both as input an
 
 ### Useful links to solve this task
 
-- https://docs.golem.network/creators/javascript/high-level/task-model/
-- https://docs.golem.network/creators/javascript/examples/hello/
-- https://docs.golem.network/creators/javascript/examples/web/
-- https://docs.golem.network/creators/javascript/examples/hashcat/
+- https://docs.golem.network/docs/creators/javascript/high-level/task-model/
```
Comment (cryptobench) [inline tasks/3-parallel-file-conversion/README.md:57]: Broken link

File: tasks/3-parallel-file-conversion/README.md [hunk 2]
```diff
@@ -54,16 +54,16 @@ base64 converted, chunked, and sent as strings within the task, both as input an
...
+- https://docs.golem.network/docs/creators/javascript/examples/hello/
```
Comment (cryptobench) [inline tasks/3-parallel-file-conversion/README.md:58]: Broken link

File: tasks/3-parallel-file-conversion/README.md [hunk 3]
```diff
@@ -54,16 +54,16 @@ base64 converted, chunked, and sent as strings within the task, both as input an
...
+- https://docs.golem.network/docs/creators/javascript/examples/web/
```
Comment (cryptobench) [inline tasks/3-parallel-file-conversion/README.md:59]: Broken link

File: tasks/3-parallel-file-conversion/README.md [hunk 4]
```diff
@@ -54,16 +54,16 @@ base64 converted, chunked, and sent as strings within the task, both as input an
...
+- https://docs.golem.network/docs/creators/javascript/examples/hashcat/
```
Comment (cryptobench) [inline tasks/3-parallel-file-conversion/README.md:60]: Broken link

File: tasks/3-parallel-file-conversion/README.md [hunk 5]
```diff
@@ -54,16 +54,16 @@ base64 converted, chunked, and sent as strings within the task, both as input an
...
 
 ### Steps:
 
 1. Convert the docker image to a [GVMI image](https://handbook.golem.network/requestor-tutorials/vm-runtime/convert-a-docker-image-into-a-golem-image) and publish it to receive an image hash
 2. Create a logic that schedules a task on Golem with image conversion, executed either from Node.js or browser context 
-    - To start, follow this article as a reference - https://docs.golem.network/creators/javascript/examples/hello/
+    - To start, follow this article as a reference - https://docs.golem.network/docs/creators/javascript/examples/hello/
```
Comment (cryptobench) [inline tasks/3-parallel-file-conversion/README.md:66]: https://docs.golem.network/docs/creators/javascript/examples/hello/

File: tasks/3-parallel-file-conversion/README.md [hunk 6]
```diff
@@ -109,8 +109,8 @@ can be found at **https://preview.golem.network/**
 
 ## Useful links:
 
-- Javascript docs: https://docs.golem.network/creators/javascript/quickstart/
-- Dapps docs: https://docs.golem.network/creators/dapps/ 
+- Javascript docs: https://docs.golem.network/docs/creators/javascript/quickstart/
```
Comment (cryptobench) [inline tasks/3-parallel-file-conversion/README.md:112]: Broken link, should be https://docs.golem.network/docs/creators/javascript/quickstarts/quickstart

**Review by cryptobench — CHANGES_REQUESTED** [review]

**Comment by jalas167** [comment]
will not patch, as mid-level api docs are no longer available, and removing the links would remove the context

**Pull request closed without merging**